Units/MMLCore/finder.pas

```diff
@@ -218,9 +218,9 @@ function ColorSame_cts2(ctsInfo: Pointer; C2: PRGB32): boolean;
     exit(false);
   if Cmax = Cmin then
   begin
-    //S and H are both zero, lets check if it mathces the tol
-    if (i.H <= (i.hueMod)) and
-       (i.S <= (i.satMod)) then
+    //S and H are both zero, the color is gray, so ignore hue and leave it to saturation tolerance.
+    if {((i.H <= i.hueMod) or  (100-i.H <= i.hueMod)) and}
```

Comment:
No need to keep the old code in comments, you can just remove the line, in my opinion.